lazyflow/operators/opPatchCreator.py

```diff
@@ -196,8 +197,8 @@ def execute(self, slot, subindex, roi, result):
             gWidth = self.GridWidth.value
             gHeight = self.GridHeight.value
 
-            numPatchesVertical = int((gHeight - pHeight) / skipVertical) + 1
-            numPatchesHorizontal = int((gWidth - pWidth) / skipHorizontal) + 1
+            numPatchesVertical = (gHeight - pHeight) // skipVertical) + 1
```

Comment:
ok. I checked in my notes. There is a review comment - "opPatchCreator.py was an experiment that was never fully finished.  It may not be possible to test it properly.". <br/>Thats why this line is untested. Thanks for spotting it though, i should fix it.